src/content/developers/docs/standards/index.md

```diff
@@ -23,9 +23,19 @@ Typically standards are introduced as [Ethereum Improvement Proposals](/eips/) (
 
 ## Types of standards {#types-of-standards}
 
-Certain EIPs relate to application-level standards (e.g. a standard smart-contract format), which are introduced as [Ethereum Requests for Comment (ERC)](https://eips.ethereum.org/erc). Many ERCs are critical standards used widely across the Ethereum ecosystem.
+There are 3 types of EIPs:
```

Comment:
```suggestion
There are 3 types of EIPs:

```
Markdown sometimes doesn't like a list right after text, so adding a new line is helpful here.